config/content.d/developer.rackspace.com.json

```diff
@@ -29,9 +29,9 @@
           "/docs/cloud-feeds/v1/developer-guide/": "https://github.com/rackerlabs/standard-usage-schemas",
           "/docs/dedicated-load-balancers/v2/": "https://github.com/rackerlabs/docs-dedicated-networking/",
           "/docs/glossary/": "https://github.com/rackerlabs/docs-rackspace/",
-          "/docs/private-cloud/rpc/v14/": "https://github.com/rackerlabs/docs-rpc/v14/",
-          "/docs/private-cloud/rpc/v13/": "https://github.com/rackerlabs/docs-rpc/v13/",
-          "/docs/private-cloud/rpc/v12/": "https://github.com/rackerlabs/docs-rpc/v12/",
+          "/docs/private-cloud/rpc/v14/": "https://github.com/rcbops/docs-rpc/v14/",
```

Comment:
The value in this key:value pair should be the exact github repo plus the branch (if it's not master). So, `https://github.com/rcbops/privatecloud-docs/v14` here, with similar changes below.